accessibility_api/accessibility_lib/wrappers/IAccessible.py

```diff
@@ -0,0 +1,200 @@
+'''
```

Comment:
Lets make a getter names is_simple_element that checks both if the target has a flag or if it's in the dictionary of traversed simple elements.